### DISCUSSION TITLE
For AI training in NanKai University, PRC on Dec 4th, 2022

### DISCUSSION THREAD

**Discussion by zhuo-yoyowz** [openvinotoolkit/openvino_notebooks#686] [Show and tell]
Hi all, please leave your inference results and questions below. Thank you~

**Comment by NkuPhyHenry** [comment]
![捕获](https://user-images.githubusercontent.com/50477265/205479221-47364454-7e5b-4d82-9505-c22f347c15a5.PNG)

**Reply by intelyoungway** [reply]
Fantastic!

**Reply by raymondlo84** [reply]
WOW!

**Comment by MITNKU** [comment]
![image](https://user-images.githubusercontent.com/64098780/205479291-35d0e52e-423e-476c-8913-d5f7ce9cde8d.png)

**Reply by zhuo-yoyowz** [reply]
Cool~

**Reply by intelyoungway** [reply]
Good work!

**Comment by MITNKU** [comment]
老师可以咨询一下，就是如果支持Interl的AVX-512 指令集的至强处理器，能否运用intel的某个类似于cuda这类的驱动，实现在CPU上可以高校训练模型，比较好奇目前Intel支持的高效模型训练的方法。

**Reply by intelyoungway** [reply]
OpenVINO最重要的作用是模型部署，以期达到最优的推理性能；
模型训练相关优化可以借助Intel oneAPI等相关工具进行尝试：比如这个 https://github.com/IntelAI/models/tree/master/quickstart/image_recognition/pytorch/resnet50/training/cpu

**Reply by MITNKU** [reply]
好的，谢谢老师讲解~

**Comment by qianqiancome** [comment]
![image](https://user-images.githubusercontent.com/66160056/205480454-92d1c915-f8e9-454c-a964-df29a2677c57.png)

**Reply by qianqiancome** [reply]
Run on my cpu~

**Reply by intelyoungway** [reply]
Nice work! You mastered the major point of AUTO-Device part:
1 throughput hint mode always focus on maximizing throughput;
2 while latency hint mode always focus on real-time!
